app/views/home/dashboard.scala.html

```diff
@@ -153,76 +153,83 @@ <h2 class="heading-medium">2. Take the online tests</h2>
                     <div class="grid grid-2-3">
                         <div class="inner-block-padr">
                             <h2 class="heading-medium">3. Attend an assessment centre</h2>
-                                @if(ConfirmedAllocatedCandidateRole.isAuthorized(usr)) {
-                                    @allocationDetails.map { aDetails =>
-                                        <p>Your assessment is booked for</p>
-                                        <p>
-                                            @dateTime(aDetails.attendanceDateTime) at @aDetails.location @aDetails.venueDescription
-                                            <br><a href="https://www.gov.uk/government/publications/fast-track-assessment-centre-locations" class="font-small" rel="external" target="_blank">See the locations</a>
-                                        </p>
-                                        <p>
-                                            If you're unable to attend, <a href="@routes.ApplicationController.helpdesk()" class="">let us know</a>.
-                                        </p>
-                                        <p><a href="https://www.gov.uk/guidance/civil-service-fast-track-apprenticeship-how-to-apply#assessment-day" rel="external" target="_blank">
-                                            Find out more about the assessment day</a></p>
-                                    }
+                                @if(FastPassReceivedRole.isAuthorized(usr)) {
```

Comment:
@Lansalo has some dashboard refactor work that we should get checked in before it gets too stale that goes some way to relieving this.